spec/dry/auto_inject_spec.rb

```diff
@@ -557,6 +557,39 @@ def initialize(other)
           expect(instance.three).to eq 3
         end
       end
+
+      context "autoinject in class and parent with regular argument in super class" do
+        let(:super_klass) do
+          Class.new do
+            attr_reader :other
+
+            def initialize(other)
+              @other = other
+            end
+          end
+        end
+
+        let(:parent_klass) do
+          Class.new(super_klass) do
+            include Test::AutoInject.kwargs[:one, :two]
+          end
+        end
+
+        let(:child_class) do
+          Class.new(parent_klass) do
+            include Test::AutoInject.kwargs["namespace.three"]
+          end
+        end
+
+        it "works" do
+          instance = child_class.new(:other)
+
+          expect(instance.other).to eq :other
+          expect(instance.one).to eq 1
+          expect(instance.two).to eq 2
+          expect(instance.three).to eq 3
+        end
```

Comment:
What's the expected behavior when you actually pass in kwargs to the constructor?

Reply:
Do you mean what is the expected behaviour if the `super_klass` in this test accepts kwargs? I suspect that is probably still broken as it was before this change. It would be nice to get this working with kwargs too but I can't think of a good way to achieve that without checking every single argument the constructor accepts and attempting to match each one before calling `super`.

Reply:
I think we should raise an error when you include auto-injector module and the super constructor is not compatible. WDYT?

Reply:
I agree.  Loud and noisy is good.